### PR TITLE
Dynamically get host IP

### PR DESCRIPTION
**Pull request by CelsoSantos**

### CODE REVIEW

File: couchbase/docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "2"
 services:
   db:
-    image: arungupta/couchbase
+    image: CelsoSantos/couchbase
```
Comment (arun-gupta) [inline couchbase/docker-compose.yml]: How is `CelsoSantos/couchbase` different from `arungupta/couchbase`?

**Comment by arun-gupta** [comment]
Do you also have the steps to create a Couchbase cluster on Kubernetes?

**Comment by bdangit** [comment]
in worker-service.yml, line https://github.com/arun-gupta/docker-images/pull/9/files#diff-3babe52f94acf7e05c41f4da77e2acf1L23  `true` value must be wrapped with quotes `'true'`.  Kube's validator keeps screaming that `env` elements must all be string.

```
$ kubectl create -f worker-service.yml
error validating "worker-service.yml": error validating data: expected type string, for field spec.template.spec.containers[0].env[2].value, got bool; if you choose to ignore these errors, turn validation off with --validate=false
```

----------
On some what of a related point, there's something about these pod/service specs that doesn't seem right.  Should the Master and Worker nodes be all within the same Couchbase K8s Service?  When we expose the couchbase service, which one would we choose?  If we choose the master thats a single point of failure.  BTW, I make no claim at knowing how to setup a couchbase cluster.  So maybe this is okay.  8)

File: couchbase/docker-compose.yml (hunk 1 above)
Comment (CelsoSantos) [inline couchbase/docker-compose.yml]: It's not. I don't recall including that on the PR version, but it should stay with arungupta/couchbase

Comment (arun-gupta) [inline couchbase/docker-compose.yml]: Most of the files seems to use that image. Can you update the PR to revert back to `arungupta/couchbase`?

**Pull request closed without merging**